package.json

```diff
@@ -13,19 +13,15 @@
     "dev:web": "npm-run-all --parallel dev:web:*",
     "dev:node": "node ./src/node/command.mjs ",
     "dev": "npm run dev:web",
-    "build:php:web": "cd wasm-build/php && bash webworker-build-wasm.sh",
-    "build:php:node": "cd wasm-build/php && bash node-build-wasm.sh",
+    "build:php:web": "cd wasm-build/php && bash webworker-build-wasm.sh && bash web-publish.sh",
+    "build:php:node": "cd wasm-build/php && bash node-build-wasm.sh && bash node-publish.sh",
     "build:php": "npm-run-all build:php:*",
-    "build:wp": "cd wasm-build/wordpress-data && bash prepare-wordpress.sh && bash bundle-data.sh",
-    "build:app:web": "esbuild --bundle src/web/app.mjs --outfile=dist-web/app.js --watch --external:xmlhttprequest",
```

Comment:
Good catch!

Reply:
Oh this reminds me: for my local dev branch I've added a few more tasks to `build:app:web` so that it does the same things as:

- `dev:web:app` (exists)
- `dev:web:web-worker` (without watch option)
- `dev:web:service-worker` (without watch option)
- `dev:web:html`

It looks like:

```json
"build:app:web": "esbuild --bundle src/web/app.mjs --outfile=dist-web/app.js --external:xmlhttprequest && esbuild --bundle src/web/web-worker.js --outfile=dist-web/web-worker.js --external:xmlhttprequest && esbuild --bundle src/web/service-worker.js --outfile=dist-web/service-worker.js --external:xmlhttprequest && cp src/web/*.html dist-web/"
```

Maybe later it can be split into individual tasks and ran in parallel using `npm-run-all`.

Another idea: since there are tasks named `dev:web` and `dev:node`, it might make sense to name the build tasks `build:web` and `build:node`.